modules/distribution/src/main/scripts/fluo

```diff
@@ -28,15 +28,18 @@ export cmd="$1"
 case "$cmd" in
   oracle|worker|init) export app="$2" ;;
 esac
-export FLUO_VERSION=${project.version}
+export FLUO_VERSION=1.2.0-SNAPSHOT
 
 if [ ! -f "$conf/fluo-env.sh" ]; then
   echo "fluo-env.sh must exist in $conf"
   exit 1
 fi
 source "$conf/fluo-env.sh"
 export CLASSPATH
-
+JAVA="java ${JAVA_OPTS[@]}"
+if [ -f "$FLUO_CONN_PROPS" ]; then
+  JAVA="$JAVA -Dfluo.conn.props=$FLUO_CONN_PROPS"
```

Comment:
this is a nice way to pass the conn props w/o having them in the command arguments.